template/src/project_name/accounts/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class AccountsConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
+    name = "accounts"
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** This should be `{{ project_name }}.accounts`

Because our Django application is inside a top level package, that top level package name needs to form part of the app config name. Simeon came across the same issue and reported it in https://github.com/ackama/django-template/issues/25

And because this is a template we do not know the packahe/project name ahead of time. Which is fine because `startproject` will do this for us it we use the `{{ project_name }}` template tag.  

I suspect that this will fix your CI issue.

‏